commerce-payment-web/src/main/resources/META-INF/resources/payment_method_action.jsp

```diff
@@ -36,37 +53,121 @@ CommercePaymentMethodGroupRel commercePaymentMethodGroupRel = (CommercePaymentMe
 			<portlet:param name="<%= Constants.CMD %>" value="<%= Constants.EDIT %>" />
 			<portlet:param name="redirect" value="<%= currentURL %>" />
 			<portlet:param name="commercePaymentMethodGroupRelId" value="<%= String.valueOf(commercePaymentMethodGroupRel.getCommercePaymentMethodGroupRelId()) %>" />
-			<portlet:param name="engineKey" value="<%= commercePaymentMethodGroupRel.getEngineKey() %>" />
+			<portlet:param name="engineKey" value="<%= engineKey %>" />
 		</portlet:actionURL>
 
 		<liferay-ui:icon
 			message="edit"
 			url="<%= editURL %>"
 		/>
 
-		<portlet:actionURL name="editCommercePaymentMethodGroupRel" var="setActiveURL">
-			<portlet:param name="<%= Constants.CMD %>" value="setActive" />
-			<portlet:param name="redirect" value="<%= currentURL %>" />
-			<portlet:param name="commercePaymentMethodGroupRelId" value="<%= String.valueOf(commercePaymentMethodGroupRel.getCommercePaymentMethodGroupRelId()) %>" />
-			<portlet:param name="active" value="<%= String.valueOf(!commercePaymentMethodGroupRel.isActive()) %>" />
-			<portlet:param name="engineKey" value="<%= commercePaymentMethodGroupRel.getEngineKey() %>" />
-		</portlet:actionURL>
-
 		<liferay-ui:icon
+			id='<%= "togglePaymentMethod-" + modalId %>'
```

Comment:
IDs shouldn't be dashed -- they should be mixedCase if possible; if not, just concatenate the strings together without a dash.